%!TEX root = ../template.tex
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% abstract-en.tex
%% NOVA thesis document file
%%
%% Abstract in English([^%]*)
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\typeout{NT FILE abstract-en.tex}%

Sanger sequencing is a foundational method in DNA analysis, yet its alignment results often require labor-intensive manual verification to identify anomalies and inaccuracies. This process is not only time-consuming but also prone to human error, posing significant challenges for laboratories managing high-throughput sequencing data. Addressing these limitations is essential to enhance the efficiency and reliability of genomic workflows.

This master's thesis introduces an AI-driven tool designed to automate the verification of DNA sequence alignment results generated by the ABI3730xl sequencer. By leveraging machine learning models trained on real-world data from 96-well plates, the tool identifies anomalies, detects sequence homologies, and evaluates the acceptability of results with high precision.

The proposed solution significantly improves laboratory workflows by streamlining the verification process and reducing reliance on manual oversight. This not only minimizes errors but also accelerates data processing, supporting advancements in precision genomics and biotechnology. Furthermore, the toolâ€™s ability to detect inaccurately labeled wells underscores its practical utility in high-throughput DNA analysis.

By addressing critical limitations in current practices, this work highlights the transformative potential of integrating artificial intelligence into genomic research. The tool contributes to the field by enabling more efficient and accurate DNA sequencing, paving the way for future innovations in genomics and biotechnology.

\keywords{
Sanger Sequencing \and
Artificial Intelligence \and
DNA Verification \and
Machine Learning \and
Genomic Workflows \and
High-Throughput Sequencing \and
Automation \and
Precision Genomics
}
